repo/packages/V/vamp/1/marathon.json.mustache

```diff
@@ -0,0 +1,47 @@
+{
+    "id": "/vamp/{{vamp.vamp-id}}",
```

Comment:
`/service/{{service.name}}`. Also change other fields here based on changes in config.json